### ISSUE TITLE
Edge glow effect have margin in the dialog

### ISSUE THREAD

**Issue by FranckRJ** [Gericop/Android-Support-Preference-V7-Fix#125]
I have already posted this issue here https://github.com/Gericop/ColorPicker/issues/1 but i just realised that you don't watch this repo, so i repost it here.

Up : Google SMS app
Down : an app with this library

![](https://i.imgur.com/fOuoYO9.png)

Is it possible to have the same behavior as the google app ?

**Comment by DreaminginCodeZH** [comment]
Solution:

```
android:clipToPadding="false"
android:scrollbarStyle="outsideOverlay"
```

You almost always want this when you set padding for a scrolling view, and it's just painful to see developers forgetting this as a user.

**Comment by Gericop** [comment]
@DreaminginCodeZH This wouldn't solve the problem because the padding is on the view around the `RecyclerView`. Will be fixed in the next release however.

**Comment by Gericop** [comment]
Will be released in v**27.0.2.0**.
